app/controllers/users_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
   private
 
     def user_params
-      params.require(:user).permit(:name, :email, :password,
+      params.require(:user).permit(:name, :email, :location, :password,
```

Comment:
Do we want to save the user's name as separate entities (first, last) or just as one concatenated field?